src/compiler/msvc.rs

```diff
@@ -1025,11 +1021,8 @@ where
                 writeln!(f, "{}:", dep)?;
             }
         }
-        Ok(process::Output {
-            status,
-            stdout,
-            stderr: stderr_bytes,
-        })
```

Comment:
The L974-L1032 changes are also unrelated to the functionality, pulled in from another branch by accident. I'll revert them.